### PR TITLE
Pull in all of OpenRegulatory docs

### PR DESCRIPTION
**Pull request by SimonBiggs**
I'm going to make use of the following clause within `Review of Documents` (which... [I just added for this purpose...](https://github.com/RadiotherapyAI/RadiotherapyAI/pull/60/commits/2b48914a0908a092acfabfb980309e4d81e6dfe3)):

> In some cases, especially when the document was retrieved from provided
> regulatory templates and the processes defined by the new documents are not yet
> in place within the company, it can be appropriate to approve the inclusion of
> the document and keep a record of the need to undergo follow-up change and
> review of the documents in question. This is particularly helpful in the
> scenario of initially creating a whole new set of documents based on regulatory
> templates. An appropriate record in this case would be a GitHub issue with a
> checklist for each of the documents that need subsequent further refinement and
> review.

This PR doesn't substantially change any of the documents (only changes are those required to get the docs to build without warnings or errors), instead, it pulls all of the template documents into the primary documentation, and then, I will create a checklist issue on GitHub detailing every document. I'll then step through the check list, editing and refining the documents and checking off the checklist in the issue once that document has undergone refinement and subsequent review.

---

Making sure each of these pulled in documents undergoes updating and review is being tracked in the following issue:

https://github.com/RadiotherapyAI/RadiotherapyAI/issues/62

---

So, long story short, this PR review is mostly a "tick-and-flick" PR acknowledging that I have pulled in and merged OpenRegulatory templates with the primary documentation covered by the understanding that these documents will be individually reviewed and updated in turn, in separate PRs.

### CODE REVIEW

File: docs/0-intended-use/index.md [hunk 1]
```diff
@@ -183,9 +203,10 @@ specific servers.
 
 ## Operating Principle
 
-<!-- > It's kind of a stretch to describe the "operating principle" of software. I guess this makes more sense for
-> hardware devices. In any case, I'd just generally state what sort of input goes in and what output comes
-> out, e.g. you could be processing images and returning diagnoses. -->
+<!-- > It's kind of a stretch to describe the "operating principle" of software. I
+> guess this makes more sense for hardware devices. In any case, I'd just
+> generally state what sort of input goes in and what output comes out, e.g.
+> you could be processing images and returning diagnoses. -->
```
Comment (sjswerdloff) [inline docs/0-intended-use/index.md:209]: Couldn't quite comment on the specific line of interest, but this was closest.
You mention `standard DICOM input output (either file based or DICOM network protocol based)` within the operating principle
Technically, DICOMWeb doesn't fall in to either of those, so you might want to include it

File: docs/1-qms/0-quality-manual-policy-objectives.md [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+<!-- > Whatever policy you outlined above, now you need to make it measurable by
+> defining objectives which can be tracked. Those objectives should not (only)
+> refer to the quality of your devices but the quality of your QMS and the
+> overall work of your organization. > Typical examples are: hiring excellence
+> in staff, providing best-of-class device performance, high standards of
+> customer satisfaction, etc. In a next step (see short-term goals), those are
+> narrowed down to concrete measures like for example the monthly number of
+> user complaints. -->
+
+When tracking the quality of the AI autocontouring software its
```
Comment (sjswerdloff) [inline docs/1-qms/0-quality-manual-policy-objectives.md:86]: This only addresses the quality of the "algorithm".
It does not address the quality of the software (defects, uptime, customer satisfaction related to non-functional requirements, such as performance, ease of use, ease of installation, etc.)

File: docs/1-qms/0-quality-manual-policy-objectives.md [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+
+## 3. Roles
+
+<!-- > Describe the roles of the people in your company. Typically this is done by
+> drawing an organigram (you could use draw.io for that). Or, you just use a
+> table like below. -->
+
+| Role(s)                                     | People           |
+| ------------------------------------------- | ---------------- |
+| CEO, QMO, PRRC, CTO, CPO, Software Engineer | Simon Biggs      |
```
Comment (sjswerdloff) [inline docs/1-qms/0-quality-manual-policy-objectives.md:111]: Is there a reason why you aren't putting each role on a separate line and listing your name for each one.
At some point, and certainly as a copyable example, one would expect each role to be filled by a different person, or at least not all of them filled by one person.

File: docs/2-techdoc/62304/software-development-maintenance-plan.md [hunk 4]
```diff
@@ -82,22 +85,31 @@ as software safety class A. The (external) risk control measures include:
 
 #### Programming Languages
 
+<!-- > List the languages you’ll be using, including compiler and language versions. -->
+
 | System                          | Name   | Version |
 | ------------------------------- | ------ | ------- |
 | Local DICOM server and ML model | Python | 3.9     |
```
Comment (sjswerdloff) [inline docs/2-techdoc/62304/software-development-maintenance-plan.md:92]: elsewhere you listed >=3.9, <3.12 for python... or something like that.  but here you only list 3.9

File: docs/data-protection/data-protection-statement-website.md [hunk 5]
```diff
@@ -9,7 +9,9 @@
 > It's painful - but you will have to go through the entire text and adjust it to your own company website
 > where necessary. We hope this template still hopes you with a general structure.
 
-# Website Data Protection Statement for Data Subjects Pursuant to Articles 13 and 14 of the EU General Data Protection Regulation
+# Website Data Protection Statement
+
+For Data Subjects Pursuant to Articles 13 and 14 of the EU General Data Protection Regulation
 
 Data protection is a particularly high priority for the management of \<enter company name\>. The use of the
```
Comment (sjswerdloff) [inline docs/data-protection/data-protection-statement-website.md:16]: I thought you had a macro for this (company name)

File: docs/index.md [hunk 6]
```diff
@@ -1,13 +1,58 @@
...
+- **Manufacturer's Address**: 17 Grampian Place Tatton NSW 2650 Australia
+- **Software as a Medical Device Identification**: {{device_name}} {{device_version}}
+
+### Overview
+
+Radiotherapy AI's contour recommendations product is deployed onsite as a DICOM
+server. It is designed to sit between your centre's CT scanner and your
+treatment planning system.
+
+When patients undergo their simulation images are auto-sent through to
```
Comment (sjswerdloff) [inline docs/index.md:40]: simulation, images

**Review by sjswerdloff — APPROVED** [review]

File: docs/0-intended-use/index.md (hunk 1 above)
Comment (SimonBiggs) [reply docs/0-intended-use/index.md:209]: I'm actually not using DICOMWeb at the moment...

File: docs/1-qms/0-quality-manual-policy-objectives.md (hunk 2 above)
Comment (SimonBiggs) [reply docs/1-qms/0-quality-manual-policy-objectives.md:86]: Yup, I'll pull this out into an issue to address when I am looking at this document again.

File: docs/1-qms/0-quality-manual-policy-objectives.md (hunk 3 above)
Comment (SimonBiggs) [reply docs/1-qms/0-quality-manual-policy-objectives.md:111]: Good call. I just wasn't sure how best to do this one. I'll change them onto new lines as I step through the documents.

File: docs/0-intended-use/index.md (hunk 1 above)
Comment (sjswerdloff) [reply docs/0-intended-use/index.md:209]: pity

File: docs/2-techdoc/62304/software-development-maintenance-plan.md (hunk 4 above)
Comment (SimonBiggs) [reply docs/2-techdoc/62304/software-development-maintenance-plan.md:92]: For the actual released and regulated Medical Device binary for now, it will be 3.9. That is the version that will go through CI etc. In the future I expect I'll have a `propagate` like function that updates a macro within the docs to match what is being utilised within the regulated build.

File: docs/data-protection/data-protection-statement-website.md (hunk 5 above)
Comment (SimonBiggs) [reply docs/data-protection/data-protection-statement-website.md:16]: Yup, I do. The was just me copying templates across though. There's heaps of items that I'll need to replace with macros. That will be something I'll be addressing as I go through each document in:

https://github.com/RadiotherapyAI/RadiotherapyAI/issues/62

File: docs/index.md (hunk 6 above)
Comment (SimonBiggs) [reply docs/index.md:40]: Thanks @sjswerdloff :), I've pulled this out into its own issue to address when I touch that document next :slightly_smiling_face:.

File: docs/0-intended-use/index.md (hunk 1 above)
Comment (SimonBiggs) [reply docs/0-intended-use/index.md:209]: Hehe :)

When I was using an online service I was actually utilising gRPC which was absolutely beautiful :slightly_smiling_face:. But, all reference in these docs to a cloud service will actually be going away initially, as my first version will be entirely offline.

Or, if there is an online section, it will be just for user account management and usage tracking for billing purposes.
